src/TractionRecUserMembership.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Drupal\openy_traction_rec;
+
+use Drupal\Core\Logger\LoggerChannelInterface;
+use Drupal\openy_traction_rec\TractionRecClient;
+use GuzzleHttp\Exception\GuzzleException;
+
+class TractionRecUserMembership {
+
+  /**
+   * Traction Rec Client service.
+   *
+   * @var \Drupal\openy_traction_rec\TractionRecClient
+   */
+  protected $tractionRecClient;
+
+  /**
+   * Logger channel.
+   *
+   * @var \Drupal\Core\Logger\LoggerChannel
+   */
+  protected $logger;
+
+  /**
+   * Constructors TractionRecFetcher.
+   *
+   * @param \Drupal\openy_traction_rec\TractionRecClient $traction_rec_client
+   *    The TractionRec API client.
+   * @param \Drupal\Core\Logger\LoggerChannelInterface $loggerChannel
+   *    Logger channel.
+   */
+  public function __construct(TractionRecClient $traction_rec_client, LoggerChannelInterface $loggerChannel) {
+    $this->tractionRecClient = $traction_rec_client;
+    $this->logger = $loggerChannel;
+  }
+
+  /**
+   * {@inheritdoc}
```

Comment:
The class doesn't extend or implement any class/interface. Please use interface implementation OR define proper method annotation.